### ISSUE TITLE
code review

### ISSUE THREAD

**Issue by qzhizhou** [WenjieLuo2333/TGS_SALT#2]
You have done a good job so far. It surprises me that the model achieve validation accuracy of more than 95%. Your idea of remove the convolutional layers of Unet with ResNet might give you credits. However, the testing accuracy is around 77%, much lower than the validation accuracy, which might be due to the limited data sets. 
Looking forward to your further improvement.

**Comment by WenjieLuo2333** [comment]
> You have done a good job so far. It surprises me that the model achieve validation accuracy of more than 95%. Your idea of remove the convolutional layers of Unet with ResNet might give you credits. However, the testing accuracy is around 77%, much lower than the validation accuracy, which might be due to the limited data sets.
> Looking forward to your further improvement.

That's right. But i only have that limited dataset for training. Kaggle only provide 4000 images for training and test the model on 18000 models.
